xbmc/interfaces/legacy/AddonUtils.h

```diff
@@ -52,11 +53,12 @@ namespace XBMCAddonUtils
   class GuiLock
   {
   public:
-    GuiLock();
+    GuiLock(XBMCAddon::LanguageHook* languageHook, bool offScreen);
```

Comment:
OK, will do. I thought it was more work to remove it from the code than to leave it in but I can understand that v17 has a feature freeze. I'll update the PR